<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Well404机器人管理工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="Creat">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>369</width>
           <height>492</height>
          </rect>
         </property>
         <attribute name="label">
          <string>创建新机器人</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="QQnum">
            <property name="placeholderText">
             <string>在这里输入机器人的QQ号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Superuser">
            <property name="placeholderText">
             <string>在这里输入你（或管理员）的QQ号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Port">
            <property name="placeholderText">
             <string>在这里输入端口号（留空则从10000至65535随机生成）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CreatBot">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>创建机器人</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="OpenBot">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>417</width>
           <height>492</height>
          </rect>
         </property>
         <attribute name="label">
          <string>管理机器人</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>请先选择选择您想要操作的机器人</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QComboBox" name="comboBox">
                <property name="placeholderText">
                 <string>请选择你的机器人</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="refresh">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>刷新</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>管理机器人</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="StartBot">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>一键开启机器人</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="CloseBot">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>关闭当前机器人</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>增减插件</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>仅支持由Well404提供的插件</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="AddPlugins">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>增添新插件</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="RemovePlugins">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>（开发中）</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>重制机器人运行信息</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ResetSuperuser">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>重置超级用户</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="Info">
        <property name="plainText">
         <string>请务必注意本窗口的信息！！！
本工具由Well404开发&amp;维护
如有任何问题请加群720053992联系群主。</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
